Nos/Views/AppView.swift

```diff
@@ -83,7 +83,7 @@ struct AppView: View {
                                 .tabItem { Label(Localized.notifications.string, systemImage: "bell") }
                                 .tag(Destination.notifications)
 
-                            RelayView()
```

Comment:
I think if the user logs out or changes their key in Settings the author the RelayView has a reference to won't change. I bet this is a bug with HomeFeedView too. I'll open a separate ticket for this.

Reply:
Created #128